src/main/java/it/gov/pagopa/payhub/activities/util/Utilities.java

```diff
@@ -125,6 +128,14 @@ public static OffsetDateTime toOffsetDateTimeEndOfTheDay(LocalDate localDate) {
         return endOfDay.atZone(ZONEID).toOffsetDateTime();
     }
 
+    public static OffsetDateTimeIntervalFilter toOffsetDateTimeIntervalFilterForDayBounds(LocalDateIntervalFilter localDateIntervalFilter){
+
+       return OffsetDateTimeIntervalFilter.builder()
+                .from(localDateIntervalFilter != null ? Utilities.toOffsetDateTime(localDateIntervalFilter.getFrom()): null)
```

Comment:
can you rename this method
```suggestion
                .from(localDateIntervalFilter != null ? Utilities.toOffsetDateTimeStartOfTheDay(localDateIntervalFilter.getFrom()): null)
```